stockflux-bitflux/src/assets/js/data/scottstock/historic/feedAdaptor.js

```diff
@@ -0,0 +1,54 @@
+import d3 from 'd3';
+import fcRebind from 'd3fc-rebind';
+import getScottStockData from '../../../../../../node_modules/stockflux-core/src/services/ScottStockService';
```

Comment:
Shouldn't this be 
`import getStockFluxData from 'stockflux-core/src/services/ScottStockService';`

Reply:
tech debt, will be fixed when I remove grunt